src/app/projects/page.tsx

```diff
@@ -0,0 +1,118 @@
+import { Card } from "@/components/Card";
+import { SimpleLayout } from "@/components/SimpleLayout";
+import { Project } from "@/interfaces/project";
+import { getAllProjects } from "@/lib/api";
+
+function ProjectStatus({ status }: { status: Project['status'] }) {
+  const statusConfig = {
+    'done': { label: 'Completed', className: 'bg-green-900/50 text-green-300 border-green-500/50' },
+    'in-progress': { label: 'In Progress', className: 'bg-blue-900/50 text-blue-300 border-blue-500/50' },
+    'abandoned': { label: 'Archived', className: 'bg-zinc-700/50 text-zinc-400 border-zinc-600/50' },
+  };
+
+  const config = statusConfig[status];
+  
+  return (
+    <span
+      className={`border px-2 py-0.5 rounded text-xs font-medium ${config.className}`}
+    >
+      {config.label}
+    </span>
+  );
+}
+
+function ProjectLinks({ project }: { project: Project }) {
+  const links = [];
+  
+  if (project.github_link) {
+    links.push({ url: project.github_link, label: 'GitHub', type: 'github' });
+  }
+  if (project.website) {
+    links.push({ url: project.website, label: 'Website', type: 'website' });
+  }
+
+  if (links.length === 0) return null;
+
+  return (
+    <div className="flex flex-wrap gap-2 mt-3">
+      {links.map((link, index) => (
+        <a
+          key={index}
+          href={link.url}
+          target="_blank"
+          rel="noopener noreferrer"
+          className="text-xs px-2 py-1 rounded border border-teal-500/50 text-teal-300 hover:bg-teal-500/10 transition-colors"
+        >
+          {link.label}
+        </a>
+      ))}
+    </div>
+  );
+}
+
+function ProjectCard({ project }: { project: Project }) {
+  return (
+    <Card className="md:col-span-3">
+      {project.og_image && (
+        <div className="mb-4 -mx-6 -mt-6">
+          <img
+            src={project.og_image}
+            alt={`${project.name} preview`}
+            className="w-full h-48 object-cover rounded-t-lg"
+          />
+        </div>
```

Comment:
Please position the image to the left of the card and ensure proper spacing and alignment. The other details of the project should be display next to the image on the right hand side.

In mobile, the image should be placed above the details, and with full width.